.circleci/config.yml

```diff
@@ -1,71 +1,68 @@
 version: 2
 jobs:
-  # Build and deploy- Just for master
-  build-and-deploy:
+  build:
     docker:
-      - image: circleci/node:lts
-      
-    working_directory: ~/repo
-
+      - image: circleci/buildpack-deps:buster
+    environment:
+        BUILD_IMAGE: italia/publiccode-editor-build
+        FINAL_IMAGE: italia/publiccode-editor
     steps:
       - checkout
-      # Run install 
-      - run: yarn install
-      # Run lint
+      - setup_remote_docker
+
+      - run: echo $DOCKER_PASSWORD | docker login --username italia --password-stdin
```

Comment:
`$DOCKER_PASSWORD` needs to be set in italia's CircleCI account with italia's DockerHub credentials.

Additionally, we'll need to create the `italia/publiccode-editor-build` and `italia/publiccode-editor` repositories on DockerHub.